src/bindings/python/src/pyopenvino/utils/utils.hpp

```diff
@@ -61,18 +70,18 @@ inline ov::Any py_object_to_any(const py::object& py_obj) {
             return _list.cast<std::vector<std::string>>();
 
         switch (detected_type) {
-            case PY_TYPE::STR:
-                return _list.cast<std::vector<std::string>>();
-            case PY_TYPE::FLOAT:
-                return _list.cast<std::vector<double>>();
-            case PY_TYPE::INT:
-                return _list.cast<std::vector<int64_t>>();
-            case PY_TYPE::BOOL:
-                return _list.cast<std::vector<bool>>();
-            default:
-                OPENVINO_ASSERT(false, "Unsupported attribute type.");
+        case PY_TYPE::STR:
+            return _list.cast<std::vector<std::string>>();
+        case PY_TYPE::FLOAT:
+            return _list.cast<std::vector<double>>();
+        case PY_TYPE::INT:
+            return _list.cast<std::vector<int64_t>>();
+        case PY_TYPE::BOOL:
+            return _list.cast<std::vector<bool>>();
+        default:
+            OPENVINO_ASSERT(false, "Unsupported attribute type.");
         }
-    // OV types
+        // OV types
```

Comment:
identation looks incorrect or it is strange to put comment after the code. Please correct it.
The same comment is for the code below